### PR TITLE
docs(testing): adding testing, environment, and sizing docs

### PR DESCRIPTION
**Pull request by link108**

### CODE REVIEW

File: content/en/docs/armory-admin/spinnaker-production-environments.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
+---
+title: Spinnaker Production Environments
+---
+
+# Architecture
+
+- Use MySQL compatible database engine
+  - Managed services like Aurora provide cross-region replication
+- Kubernetes 1.13+ cluster
```
Comment (link108) [inline content/en/docs/armory-admin/spinnaker-production-environments.md]: I think this is out of date, I believe we support 1.16+ ?

File: content/en/docs/armory-admin/spinnaker-monitoring-and-metrics.md [hunk 2]
```diff
@@ -0,0 +1,378 @@
+---
+title: Spinnaker Monitoring and Metrics
```
Comment (link108) [inline content/en/docs/armory-admin/spinnaker-monitoring-and-metrics.md:2]: we may want to make this not-visible from the sidebar, not sure how to do that atm

File: content/en/docs/armory-admin/spinnaker-monitoring-and-metrics.md [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+
+```
+totalTime query / 1000000 / count query
+```
+
+Indicates how long it takes for each HTTP controller to respond.
+
+
+
+![](https://paper.dropbox.com/ep/redirect/image?url=https%3A%2F%2Fpaper-attachments.dropbox.com%2Fs_F3370D1FA76E1F96016D2105E14ECC81229EF19F17062AFF1170FC3927E13E63_1584545219406_image.png&hmac=1Zc9dUcNVjd%2BDDz0LOiNo4sG%2FCHg0CzocCD8HQs1iaU%3D&width=1490)
```
Comment (link108) [inline content/en/docs/armory-admin/spinnaker-monitoring-and-metrics.md:37]:
```suggestion
![](https://paper.dropbox.com/ep/redirect/image?url=https%3A%2F%2Fpaper-attachments.dropbox.com%2Fs_F3370D1FA76E1F96016D2105E14ECC81229EF19F17062AFF1170FC3927E13E63_1584545219406_image.png&hmac=1Zc9dUcNVjd%2BDDz0LOiNo4sG%2FCHg0CzocCD8HQs1iaU%3D&width=1490)
```

**Review by 317brian — APPROVED** [review]
Approving but all the pages will be hidden until we can do a copyedit. This PR adds the pages and makes sure all the img links don't point to our internal wiki.

**Pull request merged**